pkg/codegen/templates/strict/strict-http.tmpl

```diff
@@ -1,5 +1,5 @@
-type StrictHandlerFunc = strictnethttp.StrictHTTPHandlerFunc
-type StrictMiddlewareFunc = strictnethttp.StrictHTTPMiddlewareFunc
+type StrictHandlerFunc = func(ctx context.Context, w http.ResponseWriter, r *http.Request) error
```

Comment:
I wouldn't open a PR for the core library for now. So I implemented the types, which are a breaking change, here instead.
If this gets motion I can open a PR for the core library.